src/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs

```diff
@@ -28,9 +29,16 @@ public void SslStream_StreamToStream_Authentication_Success()
                 Task[] auth = new Task[2];
                 auth[0] = client.AuthenticateAsClientAsync(certificate.Subject);
                 auth[1] = server.AuthenticateAsServerAsync(certificate);
-
+                
                 bool finished = Task.WaitAll(auth, TimeSpan.FromSeconds(3));
                 Assert.True(finished, "Handshake completed in the allotted time");
+
+                string testData = "Hello SSL";
```

Comment:
That I agree. But thought that for read wirte ,Handshake is anyways needed , in separate TC also we would need to duplicate this complete logic before Read/Write. I am fine making it another TC though ,